### PR TITLE
Fix: Write passive skills correctly

### PR DESCRIPTION
**Pull request by Sparkymod**
- Remove hardcoded passiveskill. It was passing the same skill `105000111` on every job.
- Parsing `Type` into skillmetadata. `Passive = 1 - Active  = 0` `Item = 3`
- Added `WritePassiveSkills` for each job
- Added `SkillMotion` into SkillMetadata for RegionEffects.

### CODE REVIEW

**Review by angelotadeucci — APPROVED** [review]

File: GameDataParser/Parsers/SkillParser.cs [hunk 1]
```diff
@@ -35,7 +38,10 @@ protected override List<SkillMetadata> Parse()
                         int levelValue = level.Attributes["value"].Value != null ? int.Parse(level.Attributes["value"].Value) : 0;
                         int spirit = level.SelectSingleNode("consume/stat").Attributes["sp"] != null ? int.Parse(level.SelectSingleNode("consume/stat").Attributes["sp"].Value) : 0;
                         float damageRate = level.SelectSingleNode("motion/attack/damageProperty") != null ? float.Parse(level.SelectSingleNode("motion/attack/damageProperty").Attributes["rate"].Value) : 0;
-                        skillLevels.Add(new SkillLevel(levelValue, spirit, damageRate, feature));
+                        string sequenceName = level.SelectSingleNode("motion/motionProperty") != null ? level.SelectSingleNode("motion/motionProperty").Attributes["sequenceName"].Value : "";
+                        string motionEffect = level.SelectSingleNode("motion/motionProperty") != null ? level.SelectSingleNode("motion/motionProperty").Attributes["motionEffect"].Value : "";
```
Comment (rhydev) [inline GameDataParser/Parsers/SkillParser.cs:42]: This is selecting the same node in both instances, should be moved to a separate variable

File: MapleServer2/Packets/JobPacket.cs [hunk 2]
```diff
@@ -69,5 +69,28 @@ public static Packet WriteSkills(PacketWriter pWriter, Player character)
 
             return pWriter;
         }
+
+        public static Packet WritePassiveSkills(PacketWriter pWriter, IFieldObject<Player> character)
+        {
+            // The x.Value.Learned == 1 is to filter for now, the skills by level 1 until player can be save on db.
+            List<SkillMetadata> passiveSkillList = character.Value.SkillTabs[0].SkillJob.Where(x => x.Value.Type == 1 && x.Value.Learned == 1).Select(x => x.Value).ToList();
```
Comment (rhydev) [inline MapleServer2/Packets/JobPacket.cs:76]: What are the different skill types? Is Type == 1 always a passive skill?

**Review by rhydev — CHANGES_REQUESTED** [review]

**Review by AlanMorel — APPROVED** [review]

File: MapleServer2/Packets/JobPacket.cs (hunk 2 above)
Comment (Sparkymod) [reply MapleServer2/Packets/JobPacket.cs:76]: Yes

**Review by rhydev — APPROVED** [review]

**Review by rhydev — CHANGES_REQUESTED** [review]
Please commit the updated hash

**Review by rhydev — APPROVED** [review]

**Pull request merged**